crates/viewer/re_time_panel/src/time_selection_ui.rs

```diff
@@ -271,6 +239,77 @@ pub fn loop_selection_ui(
             ui.ctx().set_dragged_id(right_edge_id);
         }
     }
+
+    paint_loop_selection(
+        time_ctrl,
+        time_ranges_ui,
+        ui,
+        timeline_rect.y_range(),
+        Rangef::new(timeline_rect.top(), time_area_painter.clip_rect().bottom()),
+        time_commands,
+    );
+}
+
+fn paint_loop_selection(
+    time_ctrl: &TimeControl,
+    time_ranges_ui: &TimeRangesUi,
+    ui: &egui::Ui,
+    top_y_range: Rangef,
+    full_y_range: Rangef,
+    time_commands: &[TimeControlCommand],
+) -> Option<()> {
+    // Use latest range to avoid frame delay
+    let selected_range = time_commands
+        .iter()
+        .rev()
+        .find_map(|c| {
+            if let TimeControlCommand::SetLoopSelection(range) = c {
```

Comment:
Could possibly check for a `RemoveLoopSelection` here to also have that not be frame delayed. But not as important as setting it